blog/2019-06-06-why-should-we-be-present-at-events.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: BlogPost
+category: blog
+title: Why should we – as the openHAB community – be present at events?
+author: Stephan Luckow
+date: '2019-06-06'
+previewimage: /uploads/190606_why_be_present.jpg
+tags:
+  - news
+excerpt: >-
+  My call to action for our community: Keep in mind (at every occasion) that we have marketing goals. Let's turn interested people into amateurs and turn amateurs into community members. Let's grow the community!
```

Comment:
@github-luckow-org could you add double-quotes (") around this part, or rephrase to remove the colon
```
  "My call to action for our community: Keep in mind (at every occasion) that we have marketing goals. Let's turn interested people into amateurs and turn amateurs into community members. Let's grow the community!"
```
That's what makes the build choke.

Reply:
double-quotes check